source/documentation/concepts/what-can-be-hosted-on-the-cloud-platform.html.md.erb

```diff
@@ -0,0 +1,23 @@
+---
+title: What can be hosted on the Cloud Platform
+last_reviewed_on: 2021-06-28
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+The MoJ Cloud Platform is for hosting "modern" applications.
+
+Ensure that your app:
+
+* can run in containers (Linux-based, Docker)
+* follows: [12 Factor App](https://12factor.net/)
+* uses backing services from AWS (which can be defined in Terraform)
+* its data is at OFFICIAL classification or [OFFICIAL-SENSITIVE](documentation/concepts/official-sensitive.html), not anything higher
```

Comment:
Checker doesn't like this link. Might be worth looking at to avoid issues merging.

Reply:
fixed